CHANGELOG.md

```diff
@@ -1,5 +1,107 @@
 # Lotus changelog
 
+# UNRELEASED
+
+# v1.23.3 / 2023-07-27
+
+This feature release of Lotus includes numerous improvements and enhancements for node operators, ETH RPC-providers and storage providers.
+
+## Highlights
```

Comment:
missing highlight this ? https://www.notion.so/Write-explainer-doc-for-SPs-183a7775406d451a8b19e3cdc2cdf5c3?d=d8df4e142b78427ab470db68877da1ba&pvs=4#abba19208e3b4420a6388df7a3a48b62

Reply:
come discussion link will on Monday

Reply:
#11098 has been merged